docs/usage/configuration/index.md

```diff
@@ -29,7 +29,7 @@ A path to a directory or an array of paths to directories of [custom rules][2].
   - A boolean value may be specified instead of the above object, and is equivalent to setting no options with default severity.
   - Any rules specified in this block will override those configured in any base configuration being extended.
   - [Check out the full rules list here][3].
-* `jsRules?: any`: Same format as `rules`. These rules are applied to `.js` and `.jsx` files.
+* `jsRules?: any | boolean`: Same format as `rules` or you can set to true to copy over all valid active rules from `rules`. These rules are applied to `.js` and `.jsx` files.
```

Comment:
"Same format as `rules` or explicit `true` to copy all valid active rules from `rules`."